sonar/common/jsonschemas/type-v1.0.0.json

```diff
@@ -2,88 +2,276 @@
   "title": "Document type",
   "type": "string",
   "enum": [
-    "coar:c_816b",
-    "coar:c_6501",
+    "coar:c_2f33",
     "coar:c_3248",
+    "coar:c_c94f",
     "coar:c_5794",
-    "coar:c_2f33",
-    "coar:c_bdcc",
-    "coar:c_7a1f",
-    "coar:c_46ec",
-    "coar:c_db06",
+    "coar:c_18cp",
+    "coar:c_6670",
+    "coar:c_18co",
+    "coar:c_f744",
+    "coar:c_ddb1",
+    "coar:c_3e5a",
+    "coar:c_beb9",
+    "coar:c_6501",
+    "coar:c_998f",
+    "coar:c_dcae04bc",
+    "coar:c_8544",
+    "non_textual_object",
+    "coar:c_8a7e",
+    "coar:c_ecc8",
     "coar:c_12cc",
-    "coar:c_18ws",
-    "coar:c_2fe3",
-    "coar:c_0640",
     "coar:c_18cc",
-    "coar:c_ecc8",
-    "coar:c_18cw"
+    "coar:c_18cw",
+    "coar:c_5ce6",
+    "coar:c_15cd",
+    "coar:c_2659",
+    "coar:c_0640",
+    "coar:c_2cd9",
+    "coar:c_2fe3",
+    "coar:c_816b",
+    "coar:c_93fc",
+    "coar:c_18ww",
+    "coar:c_18wz",
+    "coar:c_18wq",
+    "coar:c_186u",
+    "coar:c_18op",
+    "coar:c_ba1f",
+    "coar:c_18hj",
+    "coar:c_18ws",
+    "coar:c_18gh",
+    "coar:c_46ec2",
+    "coar:c_7a1f",
+    "coar:c_db06",
+    "coar:c_bdcc",
+    "habilitation_thesis",
+    "advanced_studies_thesis",
+    "other",
+    "coar:c_8042",
+    "coar:c_1843"
   ],
   "form": {
     "options": [
+      {
+        "label": "document_type_coar:c_2f33",
```

Comment:
What is the point of such labels? Why not the document type in natural language (ie Book, or even Book, coar:c_2f33)?

https://github.com/rero/developer-resources/blob/master/guidelines/json-schemas.md#enum

Reply:
Sorry, I didn't see the note in guidelines. But it causes a problem as the value stored is *coar:c_2f33* and in the detail view we have only this code to retrieve the translation. I prefer to store a more readable value, too, but @pronguen wanted explicitly the code.
Perhaps could we discuss about that tomorrow during the meeting?

Reply:
@sebastiendeleze As discussed this morning, do as it best suits your needs. Sorry. :slightly_smiling_face: